app/middleware/verifyToken.js

```diff
@@ -1,23 +1,22 @@
 const jwt = require('jsonwebtoken');
+const { StatusCodes } = require('http-status-codes');
+
 const config = require('../../config');
 
 module.exports = (request, response, next) => {
     const token = request.token || null;
 
     if (!token) {
         return response
-            .status(403)
-            .json({ auth: false, message: 'No token provided.' });
+            .status(StatusCodes.NO_CONTENT)

```

Comment:
Why NO_CONTENT? 

without token - you are not authorized, so it should be 401. 

It's the same as you would not send sessionID.